### PR TITLE
Add lag datamodel and API

### PR DESCRIPTION
**Pull request by rubensfig**

### CODE REVIEW

**Comment by jklare** [comment]
@akoepsel could you please take a look at this?

File: api/ofdpa.proto [hunk 1]
```diff
@@ -65,6 +65,12 @@ service OfdpaRpc {
 
   rpc ofdpaStgStatePortSet(openconfig_spanning_tree.Stp_Rstp_Interface_State) returns (OfdpaStatus) {}
 
+  rpc ofdpaTrunkCreate(trunk.Interface_Trunk_Description) returns (OfdpaStatus) {}
+  rpc ofdpaTrunkDelete(trunk.Interface_Trunk_Description) returns (OfdpaStatus) {}
+  rpc ofdpaTrunkPortPSCSet(trunk.Interface_Trunk_Description) returns (OfdpaStatus) {}
+
+  rpc ofdpaTrunkPortMemberSet(trunk.Interface_Trunk_Description) returns (OfdpaStatus) {}
```
Comment (KanjiMonster) [inline api/ofdpa.proto]: the actual call is named ofdpaPortTrunkGroupSet

Also it is weird that everything uses the same message if the actual ofdpa API calls take different arguments.

So this doesn't match the API of ofdpa at all.

File: trunk/trunk-interfaces.proto [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    LagType_BROADCAST = 3;
+    LagType_8023AD = 4;
+    LagType_TLB = 5;
+    LagType_ALB = 6;
+  };
+  LagType lag_type = 3;
+  uint32 member = 4;
+  uint32 lag_id = 5;
+  bool active = 6;
+}
```
Comment (KanjiMonster) [inline trunk/trunk-interfaces.proto]: shouldn't this be part of the ofdpa.proto?

File: api/ofdpa.proto [hunk 3]
```diff
@@ -65,6 +65,12 @@ service OfdpaRpc {
 
   rpc ofdpaStgStatePortSet(openconfig_spanning_tree.Stp_Rstp_Interface_State) returns (OfdpaStatus) {}
 
+  rpc ofdpaTrunkCreate(trunk.Interface_Trunk_Description) returns (OfdpaStatus) {}
+  rpc ofdpaTrunkDelete(trunk.Interface_Trunk_Description) returns (OfdpaStatus) {}
```
Comment (KanjiMonster) [inline api/ofdpa.proto]: e.g. this should be just
```
rpc ofdpaTrunkDelete(portNum) returns (OfdpaStatus) {}
```

as that is all it takes (and matches other ofdpa*Delete calls)

File: trunk/trunk-interfaces.proto (hunk 2 above)
Comment (rubensfig) [reply trunk/trunk-interfaces.proto]: Im trying not to have so many definitions in the ofdpa.proto file, since this out of sync with the ofdpa-grpc stuff

**Review by KanjiMonster — APPROVED** [review]
LGTM now

**Pull request merged**